### DISCUSSION TITLE
submitting Vaex for benchmarking?

### DISCUSSION THREAD

**Discussion by kylegilde** [vaexio/vaex#1569] [General]
Hi, I'm definitely on the hunt for an alternative to using Pandas. 

Vaex looks great, but I was wondering why H2O isn't benchmarking Vaex along with the dozen or so other dataframe libraries that they are benchmarking. Thanks!

Database-like ops benchmark
https://h2oai.github.io/db-benchmark/

**Comment by JovanVeljanoski** [comment]
Hi, 

Hehe, shouldn't you be asking the H2O folk that ;)

In all seriousness, making a PR to that repo with vaex has been on our (very long) to-do list... so i hope it happens eventually. I hope it will happen together with the next major release or close to it.
